Claudia/src/main/java/com/endava/claudia/task/four/AbstractUser.java

```diff
@@ -0,0 +1,69 @@
+package com.endava.claudia.task.four;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public abstract class AbstractUser {
+
+    private final long id;
+    private final String username;
+    private Map<String, Address> addressMap;
+
+    public AbstractUser(String username, String addressType, Address address) {
+        this.id = Math.abs(new Random().nextLong());
+        this.username = username;
+        this.addressMap = new HashMap<>();
+        addAddress(addressType, address);
+    }
+
+    public AbstractUser(String username) {
+        this.id = Math.abs(new Random().nextLong());
+        this.username = username;
+        this.addressMap = new HashMap<>();
+    }
+
+    public long getId() {
+
+        return id;
+    }
+
+    public String getUsername() {
+
+        return username;
+    }
+
+    public Map<String, Address> getAddressMap() {
+
```

Comment:
Remove empty line :) Please do this for other clasess as well!

Reply:
Done.